src/lib/components/top-nav.svelte

```diff
@@ -35,7 +39,23 @@
   data-testid="top-nav"
   class:bg-red-50={$dataEncoder.hasError && showNamespaceSpecificNav}
 >
-  <div class="flex items-center gap-2" />
+  <div class="flex items-center gap-2">
+    <Tooltip
+      text="{translate('labs-mode')} {$labsMode
+        ? translate('on')
+        : translate('off')}"
+      right
+    >
+      <button
+        aria-label="labs mode"
+        class="w-8 h-8 rounded flex items-center justify-center"
+        class:bg-gray-200={$labsMode}
+        on:click={() => ($labsMode = !$labsMode)}
+      >
+        <Icon name="labs" active={$labsMode} />
+      </button>
```

Comment:
Looks like you imported `<Button />` and `<IconButton />` but are using neither hah, can we make this an `<IconButton />`?

Reply:
Or should this just be a `<ToggleSwitch />` ?

Reply:
Because of the changes in styles for the Labs icon, I didn't want to add an iconClasses prop in IconButton. I think it's okay to use a regular old <button> in one-off things like this.